Firestore/core/src/firebase/firestore/model/document_key.h

```diff
@@ -43,6 +47,16 @@ class DocumentKey {
   /** Creates a new document key, taking ownership of the given path. */
   explicit DocumentKey(ResourcePath&& path);
 
+#if defined(__OBJC__)
+  DocumentKey(FSTDocumentKey* key)
```

Comment:
Add `// NOLINT(runtime/explicit)` because we're intentionally defining an automatic conversion.

Reply:
Done. Thank you for your careful review and providing a fix.